lib/opal/phaser/core/animation_manager.rb

```diff
@@ -1,11 +1,40 @@
 module Phaser
+  # The Animation Manager is used to add, play and update Phaser Animations. Any Game Object such as Phaser.
+  # Sprite that supports animation contains a single AnimationManager instance.
+  #
+  # @see http://docs.phaser.io/Phaser.AnimationManager.html
+
   class AnimationManager
+    include Native
+
     def initialize(sprite)
-      `new Phaser.AnimationManager`
+      @game   = `sprite.game`
+      @sprite = sprite
+
+      @native = `new Phaser.AnimationManager(#@sprite)`
```

Comment:
I think should point to the `sprite.animations` like we do with the cache:
https://github.com/orbitalimpact/opal-phaser/blob/master/lib/opal/phaser/core/cache.rb#L5

or we need to properly manually initialize our AnimationManager like this: meant to send this: https://github.com/photonstorm/phaser/blob/master/src/gameobjects/Sprite.js#L412